### PR TITLE
[FEATURE] PHP 7.2 support

### PR DESCRIPTION
**Pull request by finwe**

### CODE REVIEW

**Comment by finwe** [comment]
@oliverklee Followed contributing guidelines, but I am unable to request a review.

There are a few warnings in 7.2 build - deprecation notice about `each()` which is caused by old version of PHPUnit and DOM warnings which are present in previous builds, so I let them be.

**Comment by oliverklee** [comment]
Thanks for the PR, @finwe! :heart:  Looks generally good!

Could you also please remove the obsolete "Requirements" section from the README (as it's all covered by the composer.json now)?

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0, <=7.1.99"
+        "php": "^5.4 || ^7.0.0 || ^7.1.0 || ^7.2.0"
```
Comment (oliverklee) [inline composer.json]: To have this consistent, this should be 5.4.0, not 5.4.

Also, for the 7 versions, we need to use ~, not ^, as the ^ would also allow 7.3.0 (if I understand the operator correctly): https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

Comment (finwe) [reply composer.json]: You are correct with the caret operator. Changed accordingly.

**Comment by oliverklee** [comment]
Merged. Thanks for contributing to Emogrifier, @finwe! :cookie: :heart:

**Comment by finwe** [comment]
And thanks to you for such a swift response!

**Pull request merged**